src/agent/onefuzz-agent/src/local/generic_analysis.rs

```diff
@@ -16,13 +16,12 @@ use crate::{
 use anyhow::Result;
 use clap::{App, Arg, SubCommand};
 use storage_queue::QueueClient;
-use tokio::sync::mpsc::UnboundedSender;
 
 pub fn build_analysis_config(
     args: &clap::ArgMatches<'_>,
     input_queue: Option<QueueClient>,
     common: CommonConfig,
-    event_sender: Option<UnboundedSender<UiEvent>>,
+    event_sender: Option<flume::Sender<UiEvent>>,
```

Comment:
Only suggestion: in all the places where we use `flume::Sender`, just `use flume::Sender` at the top of the file so we don't have to qualify the name.